gatsby-plugin-s3/package.json

```diff
@@ -0,0 +1,60 @@
+{
+    "name": "gatsby-plugin-s3",
+    "version": "0.3.8",
+    "description": "Enables you to deploy your gatsby site to a S3 bucket.",
+    "main": "index.js",
+    "bin": {
+        "gatsby-plugin-s3": "bin.js"
+    },
+    "scripts": {
+        "prepublish": "npm run build",
+        "build": "tsc --project .",
+        "typecheck": "tsc --project . --noEmit",
+        "develop": "babel --extensions=\".ts\" --out-dir=. src/ -w",
+        "lint": "eslint \"src/**.ts\" -f junit -o ../reports/gatsby-plugin-s3-linter-results.xml",
+        "snyk-protect": "snyk-protect",
+        "prepare": "npm run snyk-protect"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/jariz/gatsby-plugin-s3.git"
+    },
+    "keywords": [
+        "gatsby",
+        "gatsby-plugin",
+        "aws",
+        "s3",
+        "aws-sdk"
+    ],
+    "peerDependencies": {
+        "gatsby": ">2.0.0-alpha"
+    },
+    "author": "Jari Zwarts <jarizw@gmail.com>",
+    "license": "MIT",
+    "bugs": {
+        "url": "https://github.com/jariz/gatsby-plugin-s3/issues"
+    },
+    "homepage": "https://github.com/jariz/gatsby-plugin-s3#readme",
+    "dependencies": {
+        "@babel/polyfill": "^7.8.3",
+        "@snyk/protect": "latest",
+        "@types/async": "^2.4.1",
+        "@types/node": "^12.0.2",
```

Comment:
these four should all be `devDependencies` I would think. otherwise anyone who imports this plugin imports these libs as well.

Reply:
Replacing `@babel/polyfill` is outside of the scope of this PR. Moving it to devDependencies causes ESLint to complain with `org.eslint.import/no-extraneous-dependencies`. A problem for another time.

Moved the others though, thanks.